tests/unit/sagemaker/image_uris/test_xgboost.py

```diff
@@ -78,7 +77,7 @@
 
 @pytest.mark.parametrize("xgboost_framework_version", XGBOOST_FRAMEWORK_CPU_GPU_VERSIONS)
 def test_xgboost_framework(xgboost_framework_version):
-    for region in regions.regions():
+    for region in FRAMEWORK_REGISTRIES.keys():
```

Comment:
If we are removing `regions.region` what are we testing here now? Seems to me we are testing `image_uris.retrieve()` to `expected_uris.framework_uri()` which is unnecessary.